smoke-tests/apps/SamplingOverrides/src/smokeTest/resources/applicationinsights.json

```diff
@@ -38,26 +38,6 @@
           "percentage": 100
         }
       ]
-    },
-    "instrumentationKeyOverrides": [
```

Comment:
We could rename this smoke tests to "OverridesTest" to cover everything, like sampling, rolenames, instrumenationKey, connectionString. 3 more smoke tests vs. one. What are the benefits of breaking them up?

Reply:
like one smoke test setup, but we can have multiple test classes for each feature?

Reply:
by breaking them into separate modules they will run in parallel in github